.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run format
+git add .
```

Comment:
Instead of doing this you can just configure lint-staged to format only the files that were changed as it will git stage them back automatically

Reply:
Oh yeah this is definitely not something we want to do. Commits should be exclusive to specific files referenced by the user.